### DISCUSSION TITLE
How can I style an Activity (beta?) goto line

### DISCUSSION THREAD

**Discussion by xenoterracide** [plantuml/plantuml#1219] [Q&A]
I've got a `goto` statement, and it works, but it's this little thin red line that overlaps other lines, and I'd like to change it's style, maybe the size of the end dot, maybe shift it to the side so it's not overlapping. I can't find any way at all to style it though. Even if I'm not able to do everything I'd like, is there anything I can do?

**Comment by The-Lum** [comment]
See similar request here:
- https://forum.plantuml.net/17445/how-to-change-colour-on-new-activity-diagram-goto

**Comment by The-Lum** [comment]
Hello @xenoterracide

With this first attempt:
- #2198

Then from last version [v1.2025.3](https://github.com/plantuml/plantuml/releases/tag/v1.2025.3), the goto lines take the color of the other lines, like:
>[![](https://img.plantuml.biz/plantuml/svg/SoWkIImgAStDuUNAJ5FGy2yeoSpFKt3KL2Z9ICrJqCZBr-JbKW22dCIar1o5ue9u80DP14WQub9X5M7DovMQKvoge56PMWmr2CaM9vWj25di1e50Udv9Ff9j46FXpWDR5saD9A7IUIcPQK3B8PG3a0Pw4G00)](https://editor.plantuml.com/uml/SoWkIImgAStDuUNAJ5FGy2yeoSpFKt3KL2Z9ICrJqCZBr-JbKW22dCIar1o5ue9u80DP14WQub9X5M7DovMQKvoge56PMWmr2CaM9vWj25di1e50Udv9Ff9j46FXpWDR5saD9A7IUIcPQK3B8PG3a0Pw4G00)

And if you want to see the `end dots`, you can add `!option debug true`, like:
```puml
@startuml
<style>
goto {LineColor red}
</style>
!option debug true

if (Option A) then (no)
    label sp_lab
    label lab1
    label sp_lab
    :X;
else (yes)
    if (Option B) then (yes)
        label sp_lab
        goto lab1
    else (no)
        :Y;   
    endif
endif
@enduml
```
>[![](https://img.plantuml.biz/plantuml/svg/TP0z3u8m48Rt9lqFvmQJSHLZK5OJLvqCX0ERr1vfZu4O_xid7yAWDzmrVPhtQPi6pZst3wlLBd1dSQzLJKpmF1c76LdouB5y2LvFV4KD6t9GOj7MmBv5hRGo5KJd0HnYu3iwY1p5Me6KpGkq49gRB9OxqZV_JcmlYLPe0qBKOHY35fRZP9dfxw1FzM-QRKFiztgzxff87x4hJINoOQGovOVU)](https://editor.plantuml.com/uml/TP0z3u8m48Rt9lqFvmQJSHLZK5OJLvqCX0ERr1vfZu4O_xid7yAWDzmrVPhtQPi6pZst3wlLBd1dSQzLJKpmF1c76LdouB5y2LvFV4KD6t9GOj7MmBv5hRGo5KJd0HnYu3iwY1p5Me6KpGkq49gRB9OxqZV_JcmlYLPe0qBKOHY35fRZP9dfxw1FzM-QRKFiztgzxff87x4hJINoOQGovOVU)

**_Enjoy,_**
Regards,
Th.

_[FYI @arnaudroques]_
